src/rest-interfaces/oauth-types.ts

```diff
@@ -12,3 +14,7 @@ export type UsersGetAuthenticatedResponse = {
 	email: string;
 	login: string;
 };
+
+export type OrganizationsResponse = {
+	orgs: Array<Octokit.AppsListInstallationsForAuthenticatedUserResponseInstallationsItem>;
```

Comment:
Can you type this interface so that it doesn't relies on the `Octokit`? This interface type should be something abstract between ui and backend. And UI doesn't have Octokit installed.

Reply:
I mean let's not installed the octkit in frontend.

Reply:
Added new types